shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -317,12 +326,29 @@ public boolean performAction(int virtualViewId, int action, Bundle arguments) {
 
                 return true;
             }
-            // TODO(goderbauer): Use ACTION_SHOW_ON_SCREEN from Android Support Library after
-            //     https://github.com/flutter/flutter/issues/11099 is resolved.
-            case 16908342: { // ACTION_SHOW_ON_SCREEN, added in API level 23
+            case ACTION_SHOW_ON_SCREEN: {
                 mOwner.dispatchSemanticsAction(virtualViewId, Action.SHOW_ON_SCREEN);
                 return true;
             }
+            case AccessibilityNodeInfo.ACTION_SET_SELECTION: {
+                final Map<String, Integer> selection = new HashMap<String, Integer>();
+                final boolean clearSelection = arguments == null
+                    || !arguments.containsKey(
+                            AccessibilityNodeInfo.ACTION_ARGUMENT_SELECTION_START_INT)
+                    || !arguments.containsKey(
+                            AccessibilityNodeInfo.ACTION_ARGUMENT_SELECTION_END_INT);
```

Comment:
Optional ultra-nit: I wonder if this would be a teensy bit clearer if you demorgan's-law'ed this as:
`hasSelection = arguments != null && arguments.containsKey(...) && arguments.containsKey(...);`

Reply:
Done.